### PR TITLE
bug1497291 bugs to relops, and cc

### PR DESCRIPTION
**Pull request by davehouse**
* Send "host unavailable" bugs to relops instead of dcops
* Set bug cc (Mark on win, Dave on lin/osx)

### CODE REVIEW

File: modules/roller/templates/dev/env.erb [hunk 1]
```diff
@@ -5,35 +5,36 @@ PYTHONDONTWRITEBYTECODE=1
...
-NOTIFY_EMAIL=relops-hardware-controller@mozilla.com
+VALID_WORKER_ID_REGEX=^.*(MS|ms|yosemite)-.*
 
-# RelOps HW Creds
 BUGZILLA_API_KEY=<%= @bugzilla_api_key %>
 BUGZILLA_URL=https://bugzilla-dev.allizom.org
 BUGZILLA_REOPEN_STATE=UNCONFIRMED
 # Change bug templates from default because bugzilla-dev does not have the "CIDuty" component
-BUGZILLA_WORKER_TRACKER_TEMPLATE={"product": "Infrastructure & Operations", "component": "RelOps", "summary": "${hostname} problem tracking", "alias": "${alias}", "version": "unspecified", "assigned_to": "NOTIFY_EMAIL", "api_key": "${api_key}"}
+BUGZILLA_WORKER_TRACKER_TEMPLATE={"product": "Infrastructure & Operations", "component": "RelOps", "summary": "[${DC}] ${alias} problem tracking", "alias": "${alias}", "version": "unspecified", "assigned_to": "NOTIFY_EMAIL", "api_key": "${api_key}"}
```
Comment (Akhliskun) [inline modules/roller/templates/dev/env.erb]: `"alias": "${alias}"` - Won't this fail if we already have a bug with an existing alias? 
A lot of them are locked with alias in CiDuty's bugzilla component.

Comment (davehouse) [reply modules/roller/templates/dev/env.erb]: This is going over to bugzilla-dev. On there, there is no CIDuty and so I have it going to RelOps for testing.

File: modules/roller/templates/prod/env.erb [hunk 2]
```diff
@@ -4,27 +4,30 @@ PYTHONDONTWRITEBYTECODE=1
...
 REBOOT_METHODS=ssh_reboot,snmp_rebootdelay,ipmi_reset,ipmi_cycle,ipmi_on,file_bugzilla_bug
 
 VALID_WORKER_ID_REGEX=^.*(MS|ms|yosemite)-.*
 
+BUGZILLA_API_KEY=<%= @bugzilla_api_key %>
+BUGZILLA_URL=https://bugzilla.mozilla.org
+BUGZILLA_REBOOT_TEMPLATE={"api_key":"${api_key}","product":"Infrastructure & Operations","component":"RelOps: Hardware","cc":"${cc}","summary":"[${DC}] ${alias} is unreachable","version":"unspecified","description":"Reboot ${hostname} ${ip}\nRequested by ${client_id}\nRelops controller action failed:${log}","blocks":"${blocks}"}
+
 NOTIFY_EMAIL=relops-hardware-controller@mozilla.com
+NOTIFY_IRC_CHANNEL=#roller
```
Comment (davehouse) [inline modules/roller/templates/prod/env.erb]: Here's where it goes to #roller for production

File: modules/roller/templates/dev/env.erb (hunk 1 above)
Comment (Akhliskun) [reply modules/roller/templates/dev/env.erb]: Thanks for the clarification!

**Pull request merged**